firebase/README.md

```diff
@@ -0,0 +1,62 @@
+## How to send web push notifications to a mobile device with Firebase
+
+This example project shows you how to send TalkJS's web push notifications to a mobile device with Firebase.
+
+> [!TIP] > [Download this example project as a zip file](https://github.com/talkjs/talkjs-examples/releases/latest/download/firebase.notifications-example.zip)
+
+## Prerequisites
+
+To run this example project, you need:
+
+- The [Firebase CLI](https://firebase.google.com/docs/cli#install_the_firebase_cli)
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+
+## How to run the example
+
+1. Clone or [download this project](https://github.com/talkjs/talkjs-examples/releases/latest/download/firebase.notifications-example.zip).
+1. In the root directory of the project, run:
+   ```sh
+   firebase login
+   ```
+   if you are not already logged in, and log in to your account
+1. Set up Firebase hosting:
+   ```sh
+   firebase init hosting
```

Comment:
What's up with the numbering? Everything is 1

Reply:
Yeah, common convention writing lists in markdown so you don't have to rewrite the numbers every time you reorder the list

Reply:
TIL.